test/test.py

```diff
@@ -106,6 +107,25 @@ def test_one_bundle(self):
 
     def _assert_bundle(self, uuid, version, manifest, metadata_files, age_range, has_specimens=True):
         bundle = Bundle(uuid, version, manifest, metadata_files)
+        expected_disease_values = {'b2216048-7eaa-45f4-8077-5a3fb4204953': {'subcutaneous melanoma'},
+                                   '3e7c6f8e-334c-41fb-a1e5-ddd9fe70a0e2.2018-09-20T232924.687620Z': {'glioblastoma'},
+                                   '3e7c6f8e-334c-41fb-a1e5-ddd9fe70a0e2': {'glioblastoma'},
+                                   '859a8bd2-de3c-4c78-91dd-9e35a3418972.2018-09-20T230221.622042Z': {'glioblastoma'},
+                                   '70184761-70fc-4b80-8c48-f406a478d5ab.2018-09-05T182535.846470Z': {'glioblastoma'}}
+        biomaterials = bundle.biomaterials.values()
+        full_uuid = f"{uuid}{f'.{version}' if version else ''}"
+        if full_uuid in expected_disease_values.keys():
+            actual_diseases = set(chain(*[getattr(bm, 'diseases', {})
```

Comment:
What's the getattr for? `diseases` should always be set to a set.

Reply:
I used `getattr` to prevent an AttributeError from being thrown.

Reply:
Why would an attribute error be thrown?

Reply:
I added it in case a biomaterial without the `diseases` attribute somehow gets added to the set. But I realized that won't happen because the `isinstance(...)` if-statement prevents that from happening.